### PR TITLE
Implemented methods in Authenticator.java

### PR DESCRIPTION
**Pull request by jasonSchroeder89**
Implemented methods in Authenticator.java.

### CODE REVIEW

File: src/main/java/com/revature/personalfinance/service/Authenticator.java [hunk 1]
```diff
@@ -3,32 +3,65 @@
...
+		// Initialize firebase config.
+		AuthenticatorUtils.firebaseInitialize();
+
 		// authenticates provided token with Firebase
-		return true;
+		FirebaseToken decodedToken = null;
+		String uid = null;
+		try {
+			decodedToken = FirebaseAuth.getInstance().verifyIdToken(jwt);
+			System.out.println(decodedToken);
```
Comment (GattieTime) [inline src/main/java/com/revature/personalfinance/service/Authenticator.java]: either remove sysout or replace with logger.

File: src/main/java/com/revature/personalfinance/service/Authenticator.java [hunk 2]
```diff
@@ -3,32 +3,65 @@
...
-		//TODO implement this method
+		// Initialize firebase config.
+		AuthenticatorUtils.firebaseInitialize();
+
+		// authenticates provided token with Firebase
+		FirebaseToken decodedToken = null;
+		String username = null;
+		try {
+			decodedToken = FirebaseAuth.getInstance().verifyIdToken(jwt);
+			System.out.println(decodedToken);
```
Comment (GattieTime) [inline src/main/java/com/revature/personalfinance/service/Authenticator.java]: sysout

**Review by GattieTime — CHANGES_REQUESTED** [review]

File: src/main/java/com/revature/personalfinance/service/Authenticator.java (hunk 1 above)
Comment (jasonSchroeder89) [reply src/main/java/com/revature/personalfinance/service/Authenticator.java]: Sysout will be removed and re-submitted.

File: src/main/java/com/revature/personalfinance/service/Authenticator.java (hunk 2 above)
Comment (jasonSchroeder89) [reply src/main/java/com/revature/personalfinance/service/Authenticator.java]: See above.

**Review by GattieTime — APPROVED** [review]

**Pull request merged**